code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -962,3 +962,26 @@ datum/reagent/medicine/syndicate_nanites/on_mob_life(mob/living/M)
 	M.adjustCloneLoss(-3*REM)
 	..()
 	return
+
+/datum/reagent/medicine/haloperidol
+	name = "Haloperidol"
+	id = "haloperidol"
+	description = "Increases depletion rates for most stimulating/hallucinogenic drugs by 5. Reduces druggy effects and jitteriness. Severe stamina regeneration penalty, causes drowsiness. 20% chance of +1 BRAIN."
```

Comment:
Yeah, we don't want gamey description. Also the text says "Severe stamina regeneration penalty" but the code actually reduce stamina damage.